### PR TITLE
add Toast to grid section

### PR DESCRIPTION
**Pull request by dineshkhadka**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,6 +73,7 @@
 - [Sass Flexible Grid System](http://flexible.gs/install/sass) - Sass flexible grid system.
 - [Susy](https://github.com/oddbird/susy) - Responsive layout toolkit for Sass.
 - [Avalanche](http://colourgarden.net/avalanche) - Lightweight, responsive, Sass-based, BEM-syntax grid system.
+- [Toast Grid](http://daneden.github.io/Toast/) - A flexible and lightweight grid framework by the creator of animate.css
```
Comment (tandavas) [inline README.md]: - Can you please change the link title to be `[Toast](...) - Description here.` to follow its original name?
- Can you please end the description sentence with a full stop as stated in [contributing.md](https://github.com/Famolus/awesome-scss/blob/master/contributing.md)?

**Comment by dineshkhadka** [comment]
Sorry if I screwed something up, I'm still new to contributions 😄

**Comment by herinkc** [comment]
@dineshkhadka No problem! There's first time for everyone 😁 

What we actually meant was to remove the word `Grid` in the beginning, as in `[Toast]` is enough since it's already in the **Grid** category.

Also, the first description was perfect. It's not necessary to state that Toast is Toast. Best to get to the point right away!

**Comment by herinkc** [comment]
Perfect! Thank you very much 🎉

**Comment by tandavas** [comment]
@dineshkhadka Thank you again , Dinesh! You're awesome 👍

**Pull request merged**